state/util/utils.py

```diff
@@ -1,12 +1,18 @@
 import os
 import string
 
-import sha3 as _sha3
-
-
-def sha3_256(x):
-    return _sha3.sha3_256(x).digest()
-
+# Python 3.6 introduces support for sha3_256 encryption through its own hashlib library. 3.6 also breaks
+# the sha3 external dependency, so use hashlib for Python 3.6 and up.
+try:
+     # For python 3.6+
+     from hashlib import sha3_256  # sha3_256 requires bytes
```

Comment:
I think we need `digest` here as well (you can search for the usage of this call in the code).

Reply:
What about this comment?

Reply:
I could move .digest() to line 139 (return sha3_256(to_string(seed))) so that both sha3 versions use .digest().